### ISSUE TITLE
Bug: i18n lint fails when using plugins

### ISSUE THREAD

**Issue by msayed98** [fnando/i18n-js#705]
## Description

running `i18n lint:translations` fails when using plugins in the config file with `config has unexpected keys: [:export_files] (I18nJS::Schema::InvalidError)`
config file:
```yml
translations:
  - file: app/assets/javascripts/translations.json
    patterns:
      - "*"
export_files:
  enabled: true
  files:
    - template: config/i18n.erb
      output: "%{dir}/%{base_name}.js"
```

## Software:

- Gem version: 4.2.2
- Ruby version: 2.7.6

**Comment by fnando** [comment]
Looks like a bug where lint tasks are not loading plugins. I'll take a look. Thanks for reporting it.

**Comment by fnando** [comment]
Just released v4.2.3 with this fix. Thanks!

**Comment by msayed98** [comment]
Thank you for your rapid response
